src/EnergyPlus/DaylightingManager.cc

```diff
@@ -1740,7 +1740,7 @@ namespace EnergyPlus::DaylightingManager {
         }
 
         ICtrl = Surface(IWin).activeWindowShadingControl;
-        ShType = WSC_ST_NoShade; // 'NOSHADE'
+        ShType = WinShadingType::NoShade; // 'NOSHADE'
```

Comment:
In this function and most (all?) of the functions named `FigureDayltgCoeffs*`, `ShType` is the type of shade available to be applied to a particular window surface. These are preprocessing functions that setup coefficients to be used later in the actual timestep daylighting calcs.
1. ShType here should only be one of the nine values from the old "WindowShadingControl Shading Types" list. By using the new broader `WindowShadingControlType ` enum class are we opening ourselves up for potential problems?
2. Side note to myself: This is why multiple WindowShadingControl objects applied to the same window **MUST** have the same Shading Type!! This is currently enforced, but I had naïvely thought that we could allow different types of shades to be slapped on at any time. That might work for solar, but clearly based on these functions it won't work for daylighting.

Reply:
1. Yes. Again it's the decoupling of types and status stuff. I can either (1) move the `WindowShadingControlType` to another enum class, or (2) move the `ShadeOff` and `ShadeCondOff` out of the current enums and change the entire logic, or (3) add comments... 
2. Exactly. Then it kinda aligned with what I replied above. I think it's totally weird to me. Maybe it's another reason to fix the logic in future branch/release.